server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -741,16 +750,203 @@ def _create_tsdb_tables(
-        ]
+    def list_model_monitoring_functions(
+        self,
+        labels: typing.Optional[list[str]] = None,
+        format_: str = mlrun.common.formatters.FunctionFormat.full,
+        infra_only: bool = False,
+    ) -> list[dict]:
+        """Retrieve a list of dictionaries, representing all the model monitoring functions."""
+
+        labels = labels or []
+        if infra_only:
+            # Model monitoring infrastructure functions
+            labels.append(
+                f"{mm_constants.ModelMonitoringInfraLabel.KEY}={mm_constants.ModelMonitoringInfraLabel.VAL}"
+            )
+        else:
+            # Model monitoring applications
+            labels.append(
+                f"{mm_constants.ModelMonitoringAppLabel.KEY}={mm_constants.ModelMonitoringAppLabel.VAL}"
+            )
+
         return services.api.crud.Functions().list_functions(
             db_session=self.db_session,
             project=self.project,
-            labels=model_monitoring_labels_list,
+            labels=labels,
+            format_=format_,
+            tag="*",
+        )
+
+    def function_summaries(
+        self,
+        start: typing.Optional[datetime] = None,
+        end: typing.Optional[datetime] = None,
+        names: typing.Optional[list[str]] = None,
+        labels: typing.Optional[list[str]] = None,
+        include_stats: bool = True,
+        include_infra: bool = True,
+    ) -> list[mlrun.common.schemas.model_monitoring.FunctionSummary]:
+        """
+        Retrieve a list of all the model monitoring functions with their summaries. Note that the response includes
+        both monitoring application real time functions and monitoring infrastructure functions.
+        :param start:            The start time of the statistics of the monitoring applications. Applicable
+                                 only when `include_status` is set to True. If not set, the default is 24 hours ago.
+        :param end:              The start time of the statistics of the monitoring applications. Applicable only
+                                 when `include_status` is set to True. If not set, the default is now.
+        :param names:            List of monitoring application function names to filter the response. Default is None.
+        :param labels:           List of labels to filter the response. Default is None.
+        :param include_stats:    If True, the function will include the statistics of the monitoring applications.
+                                 Currently, the statistics include the number of detections and possible detections.
+        :param include_infra:    If True, include the model monitoring infrastructure functions in the response.
+
+        :return:                A list of FunctionSummary objects, each representing a model monitoring function.
+        """
+
+        # Enrich response with infra functions
+        functino_summaries_list, base_period = self._enrich_function_summary_with_infra(
```

Comment:
```suggestion
        function_summaries_list, base_period = self._enrich_function_summary_with_infra(
```
Unless it's meant to be in Italian 🇮🇹

Reply:
Also in 7 other places.

Reply:
Grazie